pkg/core/rewards/schemas/claim_schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "RewardClaim",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "amount": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "specifier": {
+      "type": "string",
+      "description": "who's claiming the reward"
```

Comment:
nit: "Unique identifier for this claim instance, relative to the challenge ID. Typically, for rewards that can be claimed once per user, this is a user identifier."